src/django/api/views.py

```diff
@@ -941,38 +941,81 @@ def merge(self, request):
-                    'list_name': m.facility_list_item.facility_list.name,
-                    'list_description': m.facility_list_item.facility_list
-                    .description,
-                    'list_contributor_name': m.facility_list_item.facility_list
-                    .contributor.name,
-                    'list_contributor_id': m.facility_list_item.facility_list
-                    .contributor.id,
-                    'match_id': m.id,
-                }
-                for m
-                in facility.get_other_matches()
-            ]
+                facility_data['properties']['matches'] = [
+                    {
+                        'name': m.facility_list_item.name,
+                        'address': m.facility_list_item.address,
+                        'country_code': m.facility_list_item.country_code,
+                        'list_id': m.facility_list_item.facility_list.id,
+                        'list_name': m.facility_list_item.facility_list.name,
+                        'list_description': m.facility_list_item.facility_list
+                        .description,
+                        'list_contributor_name': m.facility_list_item
+                        .facility_list
+                        .contributor.name,
+                        'list_contributor_id': m.facility_list_item
+                        .facility_list.contributor.id,
+                        'match_id': m.id,
+                    }
+                    for m
+                    in facility.get_other_matches()
+                ]
+
+                return Response(facility_data)
 
-            return Response(facility_data)
+            match_id = request.data.get('match_id')
+
+            if match_id is None:
+                raise BadRequestException('Missing required param match_id')
+
+            match_for_new_facility = FacilityMatch \
+                .objects \
+                .get(pk=match_id)
+
+            list_item_for_match = match_for_new_facility.facility_list_item
+
+            new_facility = Facility \
+                .objects \
+                .create(name=list_item_for_match.name,
+                        address=list_item_for_match.address,
+                        country_code=list_item_for_match.country_code,
+                        location=list_item_for_match.geocoded_point,
+                        created_from=list_item_for_match)
+
+            match_for_new_facility.facility = new_facility
+            match_for_new_facility.confidence = 1.0
+            match_for_new_facility.status = FacilityMatch.CONFIRMED
+            match_for_new_facility.results = {
+                'match_type': 'split_by_administator',
+                'split_from_oar_id': match_for_new_facility.facility.id,
+            }
+
+            match_for_new_facility.save()
+
+            list_item_for_match.facility = new_facility
+            list_item_for_match.save()
```

Comment:
The delete and merge APIs add an item to `processing_results`, which acts as an audit trail of all the mutations that have been applied to the line item. Consider continuing that pattern here following the example of merge:

https://github.com/open-apparel-registry/open-apparel-registry/blob/95395af3c6d2a30dedc350902fe27ac5453e82cc/src/django/api/views.py#L909-L915

We would want to add a new `ProcessingAction.SPLIT_MATCH` constant.

Reply:
Sounds good! I added this in fc74dbe and the entry now shows "split_match" as the action and also indicates the "previous_facility_oar_id":

```
>>> f = Facility.objects.get(pk='CN2019178FQ1RKZ')
>>> f.c
f.check(           f.clean(           f.clean_fields(    f.contributors(    f.country_code     f.created_at       f.created_from     f.created_from_id  
>>> f.created_from
<FacilityListItem: FacilityListItem 594 - MATCHED>
>>> f.created_from.processing_results
[{'error': False, 'action': 'parse', 'started_at': '2019-06-26 18:54:46.654964', 'finished_at': '2019-06-26 18:54:46.659191', 'is_geocoded': True}, {'error': False, 'action': 'geocode', 'started_at': '2019-06-26 18:54:51.556838', 'finished_at': '2019-06-26 18:54:51.556877', 'skipped_geocoder': True}, {'error': False, 'action': 'match', 'started_at': '2019-06-26 18:54:52.275275', 'finished_at': '2019-06-26 18:55:10.561837'}, {'error': False, 'action': 'merge_facility', 'started_at': '2019-06-26 18:59:01.103755', 'finished_at': '2019-06-26 18:59:01.103755', 'merged_oar_id': 'CN2019177A7GXDQ'}, {'error': False, 'action': 'merge_facility', 'started_at': '2019-06-26 19:25:29.522594', 'finished_at': '2019-06-26 19:25:29.522594', 'merged_oar_id': 'CN2019177ZT0K3V'}, {'error': False, 'action': 'split_facility', 'started_at': '2019-06-27 01:08:15.123850', 'finished_at': '2019-06-27 01:08:15.123850', 'previous_facility_oar_id': 'CN2019177D9KY2N'}]
```